sql/postgres/migrate.go

```diff
@@ -166,28 +166,43 @@ func (s *state) dropTable(drop *schema.DropTable) {
 	})
 }
 
-// modifyTable builds and executes the queries for bringing the table into its modified state.
+// modifyTable builds the statements that bring the table into its modified state.
```

Comment:
```suggestion
// modifyTable builds the statements that carry the table into its modified state.
```

Reply:
Not a native speaker, but not sure that means the same. Am I wrong?

Reply:
cc @rotemtam